cucumber/authenticators_gcp/features/step_definitions/authn_gcp_steps.rb

```diff
@@ -1,18 +1,40 @@
-
-Given(/^I set GCP annotations to host "([^"]*)"$/) do |hostname|
-    i_have_a_resource "host", hostname
-    set_annotation_to_resource("authn-gcp/service-account-id", gcp_service_account_id)
-end
-
-Given(/^I obtain full format GCE identity token with audience claim "([^"]*)"$/) do | audience |
-    gce_identity_access_token(audience: audience)
-end
-
 Given(/^I obtain a GCE identity token in "([^"]*)" format and with audience claim value: "([^"]*)"$/) do | audience |
```

Comment:
you are not using the format. shouldn't you add also `format` to the params?

Reply:
these merges are killing me